cherrybomb-engine/src/lib.rs

```diff
@@ -60,7 +76,7 @@ pub async fn run(config: &Config) -> anyhow::Result<Value> {
     }
 }
 
-fn run_profile_info(config: &Config, oas: &OAS3_1, oas_json: &Value) -> anyhow::Result<Value> {
+fn run_profile_info(config: &Config, _oas: &OAS3_1, oas_json: &Value) -> anyhow::Result<Value> {
```

Comment:
Why is the oas suddenly unnecessary?

Reply:
no need of oas parameter  in the info profile , so I marked it unnecessary

Reply:
ok